static/templates/subscription_settings.handlebars

```diff
@@ -71,9 +71,11 @@
                     </li>
                     <li>
                         <div id="sub_push_notifications_setting"
-                          class="sub_setting_checkbox sub_notification_setting {{#unless in_home_view}}muted-sub{{/unless}}">
-                            <input id="push-notifystream-{{stream_id}}" class="sub_setting_control" type="checkbox" tabindex="-1" {{#if push_notifications_display}}checked{{/if}} {{#unless in_home_view}}disabled="disabled"{{/unless}}/>
+                          class="sub_setting_checkbox {{#if page_params.realm_push_notifications_enabled}}sub_notification_setting{{/if}} {{#unless in_home_view}}muted-sub{{/unless}} {{#unless page_params.realm_push_notifications_enabled}}control-label-disabled{{/unless}}">
+                            <input id="push-notifystream-{{stream_id}}" class="sub_setting_control" type="checkbox" tabindex="-1" {{#if page_params.realm_push_notifications_enabled}}{{#if push_notifications_display}}checked{{/if}}{{else}} disabled="disabled"{{/if}}{{#unless in_home_view}}disabled="disabled"{{/unless}}/>
```

Comment:
logic is:
if push notifications are off, remove `sub_notification_setting`. Because mobile notification setting should be disabled regardless of "Mute stream" setting.